# RESETFILTER
# Resets filter state
#
# Fields
# ------
# time_to_wait : Amount of ROS time to wait before resetting.
# filter_time  : Timestamp to reset to. If 0, gets current time.
# state        : Vectorized filter state to reset to. If empty, uses defaults.
# covariance   : Vectorized filter covariance to reset to. If empty, uses defaults.

float64 time_to_wait
time filter_time

float64[] state
float64[] covariance
---